app/src/main/java/com/github/multimatum_team/multimatum/AddDeadlineActivity.kt

```diff
@@ -78,20 +92,38 @@ class AddDeadlineActivity : AppCompatActivity() {
             TimePickerDialog.OnTimeSetListener { _, hoursOfDay, minutes ->
                 selectedDate = LocalDateTime.of(
                     selectedDate.year, selectedDate.monthValue, selectedDate.dayOfMonth,
-                    hoursOfDay, minutes)
+                    hoursOfDay, minutes
+                )
 
                 textTime.text = selectedDate.toLocalTime().toString()
             }
 
         // Prepare the datePickerDialog
-        val timePickerDialog = TimePickerDialog(this, timeSetListener,
-            selectedDate.hour, selectedDate.minute, true)
+        val timePickerDialog = TimePickerDialog(
+            this, timeSetListener,
+            selectedDate.hour, selectedDate.minute, true
+        )
 
         // Show the Dialog on the screen
         timePickerDialog.show()
 
     }
 
+    val checkBoxIdTime = mapOf<Int, Long>(
+        R.id.radio_notification_1h to Duration.ofHours(1).toMillis(),
+        R.id.radio_notification_5h to Duration.ofHours(5).toMillis(),
+        R.id.radio_notification_1d to Duration.ofDays(1).toMillis(),
+        R.id.radio_notification_3d to Duration.ofDays(3).toMillis()
+    )
+
+    private fun retrieveNotificationsTimes(): ArrayList<Long> {
```

Comment:
Nitpicking, but do you have a good reason to use `ArrayList` instead of `List` as the return type?

Reply:
well I though we might like to add/remove notification over time.